spec.html

```diff
@@ -39974,16 +39989,18 @@ <h1>Runtime Semantics: PerformPromiseAllSettled ( _iteratorRecord_, _constructor
               1. ReturnIfAbrupt(_nextValue_).
               1. Append *undefined* to _values_.
               1. Let _nextPromise_ be ? Call(_promiseResolve_, _constructor_, &laquo; _nextValue_ &raquo;).
-              1. Let _steps_ be the algorithm steps defined in <emu-xref href="#sec-promise.allsettled-resolve-element-functions" title></emu-xref>.
-              1. Let _resolveElement_ be ! CreateBuiltinFunction(_steps_, &laquo; [[AlreadyCalled]], [[Index]], [[Values]], [[Capability]], [[RemainingElements]] &raquo;).
+              1. Let _resolveSteps_ be the algorithm steps defined in <emu-xref href="#sec-promise.allsettled-resolve-element-functions" title></emu-xref>.
+              1. Let _resolveLength_ be the number of non-optional parameters of the function definition in <emu-xref href="#sec-promise.allsettled-resolve-element-functions" title></emu-xref>.
```

Comment:
Nit: the convention in the rest of this PR would suggest that the name of this should be `lengthResolve`, not `resolveLength`. For consistency, I think `resolveSteps` and `resolveLength` as well as both instances of each of `rejectSteps` and `rejectLength` should all be switched around.

(I assume you chose these names to match the existing `rejectSteps` variable, but I note that the `Await` macro uses `stepsFulfilled`, so there's not a clear precedent. I think it's worth renaming that variable so there's consistency.)

Reply:
That’s probably best left for an editorial PR that unifies this.

Reply:
You're introducing new variable names here, and those names should be consistent - so, `stepsResolve` and `lengthResolve`, not (as you have here) `resolveSteps` and `resolveLength`. And you might as well unify the existing inconsistent one while you're at it. I'm happy to do that for you if you'd prefer.

Reply:
Actually, `stepsFulfilled` and `onFulfilled` is used more.

---

`stepsResolve` and `stepsReject` are only used in <code>[CreateResolvingFunctions](https://tc39.es/ecma262/#sec-createresolvingfunctions)</code>.

Reply:
Works for me. I don't really care which convention we use as long as we're consistent throughout the spec. (The single `rejectSteps`, in `PerformPromiseAllSettled`, is still inconsistent, but we can fix it in a followup if you'd prefer to leave it alone here.)

Reply:
I thought I fixed that.